### ISSUE TITLE
Login Failing

### ISSUE THREAD

**Issue by abpai** [tals/uber.py#7]
When I try to login, it's giving me a `Max retries exceeded with url` error:

```
Traceback (most recent call last):
  File "ubercli.py", line 302, in <module>
    main()
  File "ubercli.py", line 299, in main
    cli.cmdloop()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/cmd.py", line 142, in cmdloop
    stop = self.onecmd(line)
  File "ubercli.py", line 258, in onecmd
    return func(*split_args)
  File "ubercli.py", line 42, in do_login
    token = UberClient.login(username, password)
  File "/Library/Python/2.7/site-packages/uber/client.py", line 43, in login
    response = uber_client._send_message(MessageTypes.LOGIN, params=data)
  File "/Library/Python/2.7/site-packages/uber/client.py", line 192, in _send_message
    response = self._post(self.ENDPOINT, data)
  File "/Library/Python/2.7/site-packages/uber/client.py", line 158, in _post
    response = self._session.post(endpoint, json.dumps(data), headers=self._headers)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 424, in post
    return self.request('POST', url, data=data, **kwargs)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 382, in request
    resp = self.send(prep, **send_kwargs)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 485, in send
    r = adapter.send(request, **kwargs)
  File "/Library/Python/2.7/site-packages/requests/adapters.py", line 372, in send
    raise ConnectionError(e)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='cn7.uber.com', port=443): Max retries exceeded with url: / (Caused by <class 'socket.error'>: [Errno 60] Operation timed out)
```

Any suggestions? Thanks in advance!

**Comment by tals** [comment]
Seems like Uber killed cn{2..9}.uber.com.
cn1.uber.com still works, though. Lemme fix :p

**Comment by abpai** [comment]
Awesome, thanks Tal!

**Comment by tals** [comment]
Fixed by 829fe68.

btw, you should consider using the official API: https://developer.uber.com.
It has more features and is actually supported by Uber :p

**Comment by abpai** [comment]
Thanks. Yeah, I was just looking for a quick wrapper around their API for a small script.
